Userland/Libraries/LibGUI/Application.h

```diff
@@ -120,6 +129,9 @@ class Application : public Core::Object {
     Vector<DeprecatedString> m_args;
     WeakPtr<Widget> m_drag_hovered_widget;
     WeakPtr<Widget> m_pending_drop_widget;
+
+    String m_config_domain;
+    Vector<NonnullRefPtr<GUI::Action>> m_recent_file_actions;
```

Comment:
Do we want Vector of NNRP? or NonnullRefPtrVector? 🤨

Reply:
I'm leaning towards removing NNRPVector and NNOPVector entirely since they appear to cause more confusion than benefit. :shrug: